stdlib/VERSIONS

```diff
@@ -62,6 +62,7 @@ array: 2.7-
 ast: 2.7-
 asynchat: 2.7-
 asyncio: 3.4-
+asyncio.mixins: 3.10-
```

Comment:
`stubtest` is still failing for <3.9 🤔 
https://github.com/python/typeshed/runs/4686388253?check_suite_focus=true

Reply:
@sobolevn In theory, mypy should support this: python/mypy#11069, but in practice, `asyncio.exceptions` is also ignored in the stubtest allowlists. You need to do the same with `asyncio.mixins` for now.

Reply:
python/mypy#12062